store/tikv/region_cache_test.go

```diff
@@ -519,6 +520,58 @@ func (s *testRegionCacheSuite) TestSendFailInvalidateRegionsInSameStore(c *C) {
 	c.Assert(err, IsNil)
 }
 
+func (s *testRegionCacheSuite) TestSendFailEnableForwarding(c *C) {
+	EnableForwarding = true
```

Comment:
Test cases run parallelly, will this affect other tests?

Reply:
Will it run parallelly with others even testRegionCacheSuite inherits OneByOneSuite?

Reply:
By default, each suite runs parallelly. 
After declaring OneByOneSuite, those suites in the OneByOneSuite list run one by one.
Not all the test suites are declared as OneByOneSuite.
So I'm not sure will this change make the CI non-stable.